webapp/public/app.css

```diff
@@ -177,6 +177,15 @@ main {
   position: relative;
 }
 
+.user-info {
+  margin-bottom: 20px;
+  text-align: center;
+}
+
+.user-info h6 {
+  text-decoration: underline;
```

Comment:
Not a good practice to underline text that is not a Link, maybe use bold or italic instead